### PR TITLE
Webrtc-98 || Create plain Javascript sample app for InApp Calling 

### PR DESCRIPTION
**Pull request by vishy-bandwidth**
Create plain Javascript sample app for InApp Calling

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    </form>
+
+    <div id="panels">
+        <div id="call_terminated_panel" class="panel" style="display: none;">
+            <p>Call terminated</p><p>Thank you</p>
+        </div>
+        <div id="outgoing_call_panel" class="panel" style="display: none;">
+            <fieldset>
+                <legend>calling</legend>
+                <p id="outgoing_call_user">+917973532006</p>
```
Comment (vikash-band) [inline html/index.html:27]: dont use any personal numbers

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<html lang="">
+<head>
+    <script type="module" src="../js/calling.js">
+
+
+    </script>
+</head>
+<body >
+
+    <div id="status_line" style="background-color: Gold">Call terminated: Not Found
```
Comment (vikash-band) [inline html/index.html:11]: reason code shall be same as sip response

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                <p id="outgoing_call_progress"></p>
+                <br>
+                <input id="cancel_outgoing_call_btn" type="button" value="Cancel">
+            </fieldset>
+        </div>
+        <div id="call_established_panel" class="panel" style="display: none;">
+            <div>
+                <fieldset>
+                    <legend>open call</legend>
+                    <p id="call_established_user">+197973532006</p>
```
Comment (vikash-band) [inline html/index.html:37]: same here, dont use any number

File: js/calling.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+        let missedCallParameter = 'Please add calling number in textbox';
+        guiError(missedCallParameter);
+        bw_log(missedCallParameter);
+        return;
+     }
+    guiInit();
+
+    phone.checkAvailableDevices()
+        .then(() => {
+            let caller = getParameter('caller', '+19197691662');
```
Comment (vikash-band) [inline js/calling.js:36]: use 1111111 or som random number

**Review by vikash-band — CHANGES_REQUESTED** [review]

File: html/index.html (hunk 1 above)
Comment (vishy-bandwidth) [reply html/index.html]: Update with dummy random number

File: html/index.html (hunk 3 above)
Comment (vishy-bandwidth) [reply html/index.html]: Update with dummy random number

File: js/calling.js (hunk 4 above)
Comment (vishy-bandwidth) [reply js/calling.js]: Update with dummy random number

File: html/index.html (hunk 2 above)
Comment (vishy-bandwidth) [reply html/index.html]: Done

**Review by vikash-band — APPROVED** [review]

**Pull request merged**